editor/translations/editor/ar.po

```diff
@@ -5184,6 +5184,9 @@ msgstr "شغِّلْ المشروع"
 msgid "Pause the running project's execution for debugging."
 msgstr "‌أوقِفْ قليلا \"تنفيذَ\" المشروع الحالي؛ للتنقيح."
 
+msgid "Resume the running project's execution."
+msgstr ""
```

Comment:
You shouldn't update the translations in the PR, this will be done by running a script when syncing with Weblate.